staging/https-nginx/README.md

```diff
@@ -122,7 +122,7 @@ $ curl https://104.198.1.26:30028 -k
 ...
 ```
 
-For more information on how to run this in a kubernetes cluster, please see the [user-guide](../../docs/user-guide/connecting-applications.md).
+For more information on how to run this in a kubernetes cluster, please see the [user-guide](https://kubernetes.io/docs/concepts/).
```

Comment:
this new link doesn't make sense much. maybe it was referring to the Service documentation before?